scripts/generate_test_sql/events.py

```diff
@@ -3,10 +3,10 @@
 
 
 FEEDING_EVENT_NAMES_LIST = (
-    "Breakfast",
-    "Brunch",
-    "Lunch",
-    "Dinner",
+    'Breakfast',
+    'Brunch',
+    'Lunch',
+    'Dinner',
```

Comment:
why the switch to single quotes here and in species.py?

Reply:
I think the linter asked for this (specifically flake8, I think)